packages/demo/src/index.js

```diff
@@ -27,6 +28,19 @@ const controller = Controller({
     },
     onlyHash: true
   }),
+
+  providers: [
+    FirebaseProvider({
+      config: {
+        apiKey: "AIzaSyAhFPZDaXOoqiokOhLnH-xMRWiikW6YU1s",
```

Comment:
Don't think so. I think it anyway could be extracted from bundle. But haven't explored in details.